src/SprykerEco/Zed/PunchoutCatalogs/Communication/Table/PunchoutCatalogsTransactionLogTable.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Table;
+
+use Orm\Zed\Company\Persistence\Map\SpyCompanyTableMap;
+use Orm\Zed\CompanyBusinessUnit\Persistence\Map\SpyCompanyBusinessUnitTableMap;
+use Orm\Zed\PunchoutCatalog\Persistence\Map\PgwPunchoutCatalogConnectionTableMap;
+use Orm\Zed\PunchoutCatalog\Persistence\Map\PgwPunchoutCatalogTransactionTableMap;
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogTransaction;
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogTransactionQuery;
+use Propel\Runtime\ActiveQuery\Criteria as Criteria;
+use Propel\Runtime\Collection\ObjectCollection;
+use Spryker\Service\UtilText\Model\Url\Url;
+use Spryker\Zed\Gui\Communication\Table\AbstractTable;
+use Spryker\Zed\Gui\Communication\Table\TableConfiguration;
+use SprykerEco\Zed\PunchoutCatalogs\Dependency\Service\PunchoutCatalogsToUtilDateTimeServiceInterface;
+
+class PunchoutCatalogsTransactionLogTable extends AbstractTable
+{
+    protected const STATUS_SUCCESSFUL = 'Successful';
+    protected const STATUS_UNSUCCESSFUL = 'Unsuccessful';
+
+    protected const COL_ID_PUNCHOUT_CATALOG_TRANSACTION = 'id_punchout_catalog_transaction';
+    protected const COL_TYPE = 'type';
+    protected const COL_COMPANY = 'company';
+    protected const COL_BUSINESS_UNIT = 'business_unit';
+    protected const COL_CONNECTION_NAME = PgwPunchoutCatalogConnectionTableMap::COL_NAME;
+    protected const COL_STATUS = 'status';
+    protected const COL_CREATED_AT = 'created_at';
+    protected const COL_SESSION_ID = 'connection_session_id';
+    protected const COL_ACTIONS = 'actions';
+
+    protected const URL_PARAM_ID_PUNCHOUT_CATALOG_TRANSACTION = 'id-punchout-catalog-transaction';
+
+    /**
+     * @uses \SprykerEco\Zed\PunchoutCatalogs\Communication\Controller\TransactionController::viewAction()
```

Comment:
`TransactionController:: ROUTE_PUNCHOUT_CATALOGS_CONNECTION_LIST_PAGE` has @see annotaton, please use @see | @uses in both places